src/gameplayv2/resistance/components/SeeComponent.js

```diff
@@ -0,0 +1,32 @@
+export class SeeComponent{
```

Comment:
For simplicity, I'd suggest having a separate See component for each role that can be seen. 

E.g. "SeeSpies" will be for all spies and merlin. "SeePercyInfo" will be for only Percy. "SeeTristanIsoldeInfo" will be for Tristan and Isolde. This allows us to have greater separation of concerns (so not all three types of `See` in one `See` component, and allows us to compose with these easier. E.g. It'll be much easier to have a spy who can also be Tristan and be seen by Isolde. 

Note: Tristan and Isolde are two good characters who know each other are good too. So you can see how it'll be really easy to extend this model to make a Spy Tristan in this manner.